docs/src/account/overview.md

```diff
@@ -0,0 +1,61 @@
+
+## What is the purpose of an account?
+
+In Miden's hybrid UTXO- and account-based model, accounts enable the creation of expressive smart contracts via a Turing-complete language. As examples, an account with a wallet interface can hold the assets of a user while a DEX account could have an interface to trade assets.
+
+## Account composition
+
+An `Account` is composed of several core parts, illustrated below:
+
+<p style="text-align: center;">
+    <img src="../img/account/account-definition.png" style="width:30%;" alt="Account diagram"/>
+</p>
+
+These parts are:
+
+1. [ID](id.md)
+2. [Code](code.md)
+3. [Storage](storage.md)
+4. [Vault](#vault)
+5. [Nonce](#nonce)
+
+### Vault
+
+> [!Note]
+> A collection of [assets](../asset.md) stored by the `Account`.
+
+Large amounts of fungible and non-fungible assets can be stored in the account's vault.
+
+### Nonce
+
+> [!Note]
+> A counter incremented with each state update to the `Account`.
+
+The nonce ensures that an account has a unique _commitment_ (or "hash") after every transaction, even if it contains the same assets and has the same storage state. That in turn allows ordering of transactions and prevents replay attacks. Whenever the state of an account changes in a transaction, its nonce must be incremented and it can only be incremented exactly once per transaction.
+
+Note that a transaction does not always change the state of an account. For instance, a transaction in which two SWAP notes are matched together does not necessarily change anything about the account state. Consequently, the nonce does not have to be incremented.
+
+## Account lifecycle
+
+Throughout its lifetime, an `Account` progresses through various phases:
+
+- **Creation and Deployment:** Initialization of the `Account` on the network.  
+- **Active Operation:** Continuous state updates via `Account` functions that modify the storage, nonce, and vault.
+- **Termination or Deactivation:** Optional, depending on the contract's design and governance model. Note however, that there is no functionality in the protocol to delete an account.
+
+### Account creation
+
+For an `Account` to be recognized by the network, it must exist in the [account database](../state.md#account-database) maintained by Miden node(s).
+
+However, a user can locally create a new `Account` ID before it's recognized network-wide. The typical process might be:
+
+1. Alice generates a new `Account` ID locally (according to the desired `Account` type) using the Miden client.
+2. The Miden client checks with a Miden node to ensure the ID does not already exist.
+3. Alice shares the new ID with Bob (for example, to receive assets).
+4. Bob executes a transaction against his account, creating a note with assets for Alice.
+5. Alice consumes Bob's note in a transaction against her account to claim the asset. This
+transaction is the first transaction against Alice's account and so it will register the account
+ID in the account database.
```

Comment:
"ID in the account database." is not very clear that this registered on the network, rather than locally for Alice